src/OrleansSQLUtils/Storage/Provider/RelationalStorageProvider.cs

```diff
@@ -0,0 +1,550 @@
+﻿using Orleans.Providers;
+using Orleans.Runtime;
+using Orleans.Serialization;
+using Orleans.SqlUtils;
+using OrleansSQLUtils.Storage;
+using OrleansSQLUtils.Storage.Provider;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+
+namespace Orleans.Storage
+{
+    /// <summary>
+    /// Logging codes used by <see cref="RelationalStorageProvider"/>.
+    /// </summary>
+    /// <remarks> These are taken from <em>Orleans.Providers.ProviderErrorCode</em> and <em>Orleans.Providers.AzureProviderErrorCode</em>.</remarks>
```

Comment:
I tried to avoid changing other other parts. As indicated in comments, this would need some consolidating. I'm not sure what do here, but maybe we get second opinions from @sergeybykov or @jdom.